### PR TITLE
BREAK: make `cachetory.backends.async_.from_url` non-async

### PR DESCRIPTION
**Pull request by eigenein**
Being `async`, `from_url` is inconvenient to use when initializing a cache instance. It also proved to be unnecessary, except for maybe a few corner cases.

### CODE REVIEW

File: tests/backends/async_/test_redis.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 @fixture
 async def backend() -> AsyncIterable[RedisBackend]:
-    async with await RedisBackend.from_url("redis://localhost:6379") as backend:
+    async with RedisBackend.from_url("redis://localhost:6379") as backend:
```
Comment (eigenein) [inline tests/backends/async_/test_redis.py:14]: `async with await` looks nice 😄

**Review by streeck — APPROVED** [review]
